client/src/components/Collections/common/CollectionEditView.vue

```diff
@@ -0,0 +1,260 @@
+<template>
+    <div>
+        <h4>{{ l("Edit Collection Attributes") }}</h4>
+        <b-alert show variant="warning" dismissible>
+            {{
+                l(
+                    "Collections are immutable. This means there will be some things you cannot change without creating a new collection. "
+                )
+            }}
+        </b-alert>
+        <div v-if="jobError">
+            <b-alert show variant="danger" dismissible>
+                {{ errorMessage }}
+            </b-alert>
+        </div>
+        <b-tabs content-class="mt-3">
+            <b-tab>
+                <template v-slot:title> <i class="fa fa-bars"></i>{{ l(" Attributes") }}</template>
+                <b>{{ l("Name: ") }}</b> <i>{{ collectionName }}</i>
+                <br />
+                <b>{{ l("Collection Type: ") }}</b> <i>{{ collectionType }}</i>
+                <br />
+                <b>{{ l("Elements: ") }}</b> <br />
+                <div v-for="element in collectionElements" :key="element">{{ element.element_identifier }} <br /></div>
+            </b-tab>
```

Comment:
This tab looks a little weird. I think you can remove this ? If you do want to keep it I would suggest mounting the DatasetCollectionContents.vue component (or a variant of it for this context) here or maybe adding your own component that includes the content type and name name as an item ?

Reply:
Ultimately this tab will look better with #6966, where users can change the name of the collection, annotations, etc. For now, it's really just a place holder. I can remove it for now though, since it doesn't do very much except print some information that was important for me to see what was going on.